Makefile.am

```diff
@@ -4013,6 +4013,9 @@ journalctl_LDADD += \
 	$(QRENCODE_LIBS)
 endif
 
+journalctl_SOURCES += \
+	src/journal/journal-fields.c
```

Comment:
There is a missing `src/journal/journal-fields.h`.
See https://semaphoreci.com/systemd/systemd/branches/pull-request-2159/builds/1
```
../src/journal/journalctl.c:59:28: fatal error: journal-fields.h: No such file or directory
 #include "journal-fields.h"
                            ^
compilation terminated.
../src/journal/journal-fields.c:43:28: fatal error: journal-fields.h: No such file or directory
 #include "journal-fields.h"
                            ^
compilation terminated.
```